package.json

```diff
@@ -81,7 +82,6 @@
     "glob": "^7.1.2",
     "jszip": "^3.1.5",
     "lodash": "^4.17.5",
-    "mockserver-client": "5.6.1",
```

Comment:
Are you sure this is the right change? I thought that `lambda-tools` exposed a helper that used mockserver-client directly and that it was used for more than just testing of `lambda-tools` itself

Reply:
Where do you see that? The only usage I see is here: https://github.com/lifeomic/lambda-tools/blob/de4981c4d38a8c89cb6024cd2e8f8a9428a656d1/test/mockServerLambda.test.js#L10

A `mockServerClient` instance variable is passed in [here](https://github.com/lifeomic/lambda-tools/blob/master/src/mockServerLambda.js), but this project doesn't import it directly except for tests

Reply:
I was thinking of the API you linked to. I really wish there was such a think as optional peer dependencies to clearly show what the compatible versions are.

Reply:
I agree, that pattern with no optional peer dependencies does make it hard to communicate a clear contract.